### PR TITLE
Fix network error

### PR DESCRIPTION
**Pull request by pshishko**
"NetworkError: 404 Not Found - http://example.dev/%7B%7BresourceMap.flash%7D%7D"

flash -> swf (flash not exists in resourceMap)
data -> ng-data (to prevent load not initialized data)

### CODE REVIEW

File: dist/vidBg.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ module.run(['$templateCache', function($templateCache) {
     '		<source type="video/webm">\n' +
     '		<source type="video/mp4">\n' +
     '		<source type="video/ogg">\n' +
-    '		<object type="application/x-shockwave-flash" data="{{resourceMap.flash}}">\n' +
+    '		<object type="application/x-shockwave-flash" ng-data="{{resourceMap.swf}}">\n' +
```
Comment (2013gang) [inline dist/vidBg.js:123]: I was actually copying over this part from some other sources. Have you tested your changes for flash files?

**Comment by pshishko** [comment]
Hi, I don't tested flash video, only mp4. But in js a flash resource mapped to resourceMap.swf.

`if (ele.toUpperCase().indexOf('.WEBM', ele.length - '.WEBM'.length) !== -1) {
                    resourceMap.webm = ele;
                  } else if (ele.toUpperCase().indexOf('.MP4', ele.length - '.MP4'.length) !== -1) {
                    resourceMap.mp4 = ele;
                  } else if (ele.toUpperCase().indexOf('.OGV', ele.length - '.OGV'.length) !== -1) {
                    resourceMap.ogv = ele;
                  } else if (ele.toUpperCase().indexOf('.SWF', ele.length - '.SWF'.length) !== -1) {
                    resourceMap.swf = ele;
                  }`

and

`vid.children().eq(0).attr('src', resourceMap.webm ? resourceMap.webm : '');
            vid.children().eq(1).attr('src', resourceMap.mp4 ? resourceMap.mp4 : '');
            vid.children().eq(2).attr('src', resourceMap.ogv ? resourceMap.ogv : '');
            return vid.children().eq(3).children().eq(0).attr('value', resourceMap.swf ? resourceMap.swf : '');`

**Comment by pshishko** [comment]
Thanks, can you update bower "angular-video-background" to get latest version?

**Comment by 2013gang** [comment]
Hi @pshishko, I am not able to do it in a while. Please modify it in your local file for now.

**Comment by 2013gang** [comment]
sorry for the inconvenience..

**Comment by pshishko** [comment]
Hi, all is ok, in almost no time too. I have a problems with this component but they are difficult to solve with small changes.
Problems with the order of the loading resources and with Chrome after the build scripts.
Now I decided to rewrite it all the easier. If it will - I will send a link to see.
Have a nice day, thanks for the response.

**Pull request merged**